packages/menu-bar/test/sub-menu.test.js

```diff
@@ -465,14 +469,48 @@ describe('open on hover', () => {
     expect(subMenu.listenOn).to.equal(buttons[0]);
   });
 
-  it('should close open sub-menu on mouseover on button without nested items', async () => {
+  it('should close sub-menu on mouseover on different item', async () => {
     fire(buttons[0], openOnHoverEvent);
     await nextRender(subMenu);
     fire(buttons[1], openOnHoverEvent);
     await nextRender(subMenu);
     expect(subMenu.opened).to.be.false;
   });
 
+  it('should close opened sub-menu if _requestClose is called', async () => {
+    fire(buttons[0], openOnHoverEvent);
+    await nextRender(subMenu);
+    expect(subMenu.opened).to.be.true;
+
+    menu._requestClose();
```

Comment:
Shouldn't access private API from a test case. Rather use `mouseleave` on the menu (which internally calls `_requestClose`)